File: scalaz/src/main/scala/org/openapitools/client/api/InvalidMessageErrorAllOf.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime


import InvalidMessageErrorAllOf._

case class InvalidMessageErrorAllOf (
  result: Option[AnyType],
msg: Option[AnyType],
/* The raw content of the message.  */
  rawContent: Option[String])

object InvalidMessageErrorAllOf {
  import DateTimeCodecs._

  implicit val InvalidMessageErrorAllOfCodecJson: CodecJson[InvalidMessageErrorAllOf] = CodecJson.derive[InvalidMessageErrorAllOf]
  implicit val InvalidMessageErrorAllOfDecoder: EntityDecoder[InvalidMessageErrorAllOf] = jsonOf[InvalidMessageErrorAllOf]
  implicit val InvalidMessageErrorAllOfEncoder: EntityEncoder[InvalidMessageErrorAllOf] = jsonEncoderOf[InvalidMessageErrorAllOf]
}

File: javascript-closure-angular/API/Client/InvalidMessageErrorAllOf.js
goog.provide('API.Client.InvalidMessageError_allOf');

/**
 * @record
 */
API.Client.InvalidMessageErrorAllOf = function() {}

/**
 * @type {!API.Client.AnyType}
 * @export
 */
API.Client.InvalidMessageErrorAllOf.prototype.result;

/**
 * @type {!API.Client.AnyType}
 * @export
 */
API.Client.InvalidMessageErrorAllOf.prototype.msg;

/**
 * The raw content of the message. 
 * @type {!string}
 * @export
 */
API.Client.InvalidMessageErrorAllOf.prototype.rawContent;

